Manito/Manito/Screens/Main/MainViewController.swift

```diff
@@ -321,13 +284,54 @@ final class MainViewController: BaseViewController {
     }
     
     func showRoomIdErrorAlert() {
-        makeRequestAlert(title: "해당 마니또 방의 정보를 불러오지 못했습니다.", message: "해당 마니또 방으로 이동할 수 없습니다.", okAction: nil)
+        self.makeRequestAlert(title: "해당 마니또 방의 정보를 불러오지 못했습니다.", message: "해당 마니또 방으로 이동할 수 없습니다.", okAction: nil)
```

Comment:
p3;

이 부분을 makeAlert로 만들수도 있지 않나요 ?? 그렇게 되면 굳이 okAction을 적지 않아도 될 것 같아요 !
추가적으로 TextLiteral로 분리가 된다면 좋겠네요 !

```suggestion
        self.makeAlert(title: "해당 마니또 방의 정보를 불러오지 못했습니다.",
                       message: "해당 마니또 방으로 이동할 수 없습니다.")
```

Reply:
[5b78e36](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/375/commits/5b78e369fc833d0907b44a105c6953832ed757cf)
반영했습니다.

Reply:
[499e6ca](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/375/commits/499e6ca383e5b3a73178117a88f2bb71af859ffd)
반영했습니다.